rest/bulk_api.go

```diff
@@ -458,6 +458,19 @@ func (h *handler) handleBulkGet() error {
 				base.AuditFieldDocID:      docid,
 				base.AuditFieldDocVersion: revid,
 			})
+			if atts, ok := body[db.BodyAttachments]; ok && atts != nil {
+				attsMap, ok := atts.(db.AttachmentsMeta)
+				if !ok {
+					base.WarnfCtx(h.ctx(), "Unexpected format of attachments in the body %+v", atts)
+				}
+				for attachment := range attsMap {
+					base.Audit(h.ctx(), base.AuditIDAttachmentRead, base.AuditFields{
+						base.AuditFieldDocID:        docid,
+						base.AuditFieldDocVersion:   revid,
+						base.AuditFieldAttachmentID: attachment,
+					})
+				}
+			}
```

Comment:
This is only reading metadata of the attachment, unless `includeAttachments` is set.

Does this need to be conditional?